integrated_channels/api/v1/degreed/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+    url mappings for integrated_channels/v1/degreed/
```

Comment:
nit: just noted the path here `integrated_channels/v1/degreed/` probably needs to be `integrated_channels/api/v1/degreed/`? having said this all other channels seem to be doing this style! not too worried if we keep it the way it is or even remove the comment!